cypress/pages/LoginPage.js

```diff
@@ -16,7 +16,6 @@ class LoginPage {
    */
   static login() {
     if (Cypress.env('e2e')) {
-      this.logout();
```

Comment:
was there previously a flow in the old UI to logout, and now no longer exists?

Reply:
The original tests never worked, so this wasn't required. Also, the logout with newer KC now requires an id_token_hint that (used to be a static endpoint) so you can't log out anyway.  

TLDR - this was never required and doesn't add value.